index.js

```diff
@@ -0,0 +1,50 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let booksCollection = [];
```

Comment:
- [ ] I think you forgot to save your changes in the localStorage, I can see you did a wonderful job but kindly include saving changes in the localStorage as required in the project requirements as attached in the image below.